src/crimson/os/seastore/seastore_types.h

```diff
@@ -1496,6 +1516,10 @@ struct record_t {
   record_size_t size;
   mod_time_point_t commit_time;
   record_commit_type_t commit_type;
+  extents_num_t lba_extents_num = 0;
+  extents_num_t backref_extents_num = 0;
+  extents_num_t onode_extents_num = 0;
+  extents_num_t omap_extents_num = 0;
```

Comment:
Hmm, not sure if these stats worths to be written into every record.

Is it accetable to keep them as relative values without persistence?

Reply:
> Is it accetable to keep them as relative values without persistence?

Like tree depth, we need to make these stats durable. It seems that these stats can be recalcuated when scanning backref tree on startup, I'll try that approach so that no storage space overhead is needed